api/nodes/serializers.py

```diff
@@ -127,6 +128,114 @@ def update(self, instance, validated_data):
+    filterable_fields = frozenset.union(UserSerializer.filterable_fields, local_filterable)
+
+    id = ser.CharField(source='_id')
+    fullname = ser.CharField(read_only=True, help_text='Display name used in the general user interface')
+    given_name = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    middle_name = ser.CharField(read_only=True, source='middle_names', help_text='For bibliographic citations')
+    family_name = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    suffix = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    date_registered = ser.DateTimeField(read_only=True)
+    gravatar_url = ser.CharField(read_only=True, help_text='URL for the icon used to identify the user. Relies on '
+                                                           'http://gravatar.com ')
+    employment_institutions = ser.ListField(read_only=True, source='jobs', help_text='An array of dictionaries '
+                                                                                     'representing the '
+                                                                                     'places the user has worked')
+    educational_institutions = ser.ListField(read_only=True, source='schools', help_text='An array of dictionaries '
+                                                                                         'representing the places the '
+                                                                                         'user has attended school')
+    social_accounts = ser.DictField(read_only=True, source='social', help_text='A dictionary of various social media '
+                                                                               'account identifiers including an array '
+                                                                               'of user-defined URLs')
+    links = LinksField({
+        'html': 'absolute_url',
+        'nodes': {
+            'relation': Link('users:user-nodes', kwargs={'user_id': '<_id>'})
+        },
+        'edit contributor': Link('nodes:node-contributor-detail', kwargs={'user_id': '<_id>', 'node_id': '<node_id>'})
+    })
+
+    def absolute_url(self, obj):
+        return obj.absolute_url
+
+    def create(self, validated_data):
+        user = self.context['request'].user
+        auth = Auth(user)
+        node = self.context['view'].get_node()
+        contributor = User.load(validated_data['_id'])
+        if not contributor:
+            raise NotFound('User with id {} cannot be found.'.format(validated_data['_id']))
+        elif contributor in node.contributors:
+            raise ValidationError('User {} already is a contributor.'.format(contributor.username))
+        else:
+            node.add_contributor(contributor=contributor, auth=auth, save=True)
+            contributor.node_id = node._id
+        return contributor
+
+
+class ContributorDetailSerializer(ContributorSerializer):
```

Comment:
From an API standpoint, the information should be available everywhere. The UI can determine if it wants to display the information, but we don't want specific API endpoints that are slightly different here and there.